src/renderers/WebGLRenderer.js

```diff
@@ -1434,7 +1425,7 @@ function WebGLRenderer( parameters ) {
 
 	function renderObject( object, scene, camera, geometry, material, group ) {
 
-		object.onBeforeRender( _this, scene, camera, geometry, material, group );
+		object.onBeforeRender( _this, scene, camera, _currentViewport, geometry, material, group );
```

Comment:
Is there a reason for the position of `_currentViewport` in the arguments? It might be better added to the end. I think this would qualify as a BC-break otherwise.

Reply:
Right. I don't think you should be changing the `onBeforeRender()` signature.

Reply:
see https://github.com/mrdoob/three.js/pull/13081#discussion_r160810155

I'm open to any solution. This or `.getCurrentViewport()`.

Reply:
@mrdoob What do you think? 😊

Reply:
I guess `.getCurrentViewport()` is safer yeah... 😇

Reply:
Okay, I'll cherry pick the first two commits and amend.